.gitlab-ci.yml

```diff
@@ -63,11 +63,14 @@ publish:build:master:
   stage: publish
   dependencies:
     - build
+  services:
+    - docker:dind
   script:
-    - docker load -i image.tar
-    - export COMMIT_TAG="$CI_COMMIT_REF_NAME"_"$CI_COMMIT_SHA";
-    - docker tag $DOCKER_REPOSITORY:pr $DOCKER_REPOSITORY:$COMMIT_TAG;
-    - docker push $DOCKER_REPOSITORY:$COMMIT_TAG;
+    - echo "publishing image to docker hub"
```

Comment:
Except, you're not..?

Reply:
Hehehe, correct.

That contradiction is repeated in all the other repos (check [mender-artifact](https://github.com/mendersoftware/mender-artifact/blob/master/.gitlab-ci.yml#L92) as an example) so I just wanted to be consistent.

Reply:
Lol, consistently wrong.